.pre-commit-config.yaml

```diff
@@ -20,3 +20,12 @@ repos:
     rev: v0.23.1
     hooks:
       - id: markdownlint
+
+  - repo: local
+    hooks:
+      - id: goreleaser-check
+        name: Validate goreleaser config  yaml
+        entry: goreleaser check
+        language: system
+        files: '.goreleaser.yml'
+        pass_filenames: false
```

Comment:
This business of checking the config file on merge seems like a good practice for us. We can put it in trussworks/pre-commit-hooks if we like it.